src/lib/browse/layers/CriticalIssuesLayerControl.svelte

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+  import type { GeoJSONSource, MapGeoJSONFeature } from "maplibre-gl";
+  import { map } from "../../../stores";
+  import { CollapsibleCard, ColorLegend, InteractiveLayer } from "../../common";
+  import {
+    Checkbox,
+    CheckboxGroup,
+    ErrorMessage,
+    FormElement,
+  } from "../../govuk";
+  import {
+    cleanupSource,
+    emptyGeojson,
+    overwriteCircleLayer,
+  } from "../../maplibre";
+  import { parseCriticalIssuesExcel } from "../data";
+
+  let fileInput: HTMLInputElement;
+
+  let source = "criticals";
+  let color = "red";
+  let show = true;
+  let numberIssues = 0;
+  let errorMessage = "";
+
+  cleanupSource($map, source);
+  $map.addSource(source, {
+    type: "geojson",
+    data: emptyGeojson(),
+    cluster: true,
+    clusterMaxZoom: 14,
```

Comment:
Generally following https://maplibre.org/maplibre-gl-js/docs/examples/cluster/. Default params here seem fine to me.